py/torch_tensorrt/dynamo/conversion/impl/pad.py

```diff
@@ -1,14 +1,22 @@
-import copy
 from typing import Optional, Sequence, Union
 
-import torch
-import torch_tensorrt.dynamo.conversion.impl as impl
+import tensorrt as trt
 from torch.fx.node import Target
 from torch_tensorrt.dynamo._SourceIR import SourceIR
 from torch_tensorrt.dynamo.conversion._ConversionContext import ConversionContext
-from torch_tensorrt.fx.converters.converter_utils import has_dynamic_shape
+from torch_tensorrt.fx.converters.converter_utils import (
+    get_trt_tensor,
```

Comment:
This should use the `dynamo.converters.converter_utils` implementation of `get_trt_tensor`